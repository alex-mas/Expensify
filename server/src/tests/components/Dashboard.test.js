import React from 'react';
import ReactShallowRenderer from 'react-test-renderer/shallow';
import toJSON from 'enzyme-to-json';
import {shallow} from 'enzyme';
import Dashboard from '../../components/dashboard';


describe('Header component unit tests', function(){
    test('should render header correctly',()=>{
        const wrapper = shallow(<Dashboard/>);
        expect(wrapper).toMatchSnapshot();
    });
});
