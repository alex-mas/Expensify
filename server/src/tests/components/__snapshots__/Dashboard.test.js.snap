// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component unit tests should render header correctly 1`] = `
<div>
  <Connect(ExpensesSummary) />
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
